external/fv3fit/fv3fit/train_microphysics.py

```diff
@@ -84,6 +81,20 @@ class TrainConfig:
     verbose: int = 2
     shuffle_buffer_size: Optional[int] = 100_000
 
+    @property
+    def _model(
+        self,
+    ) -> Union[models.MicrophysicsConfig, models.ConservativeWaterConfig]:
+        if self.model:
```

Comment:
Defaults are no models so it would be harder to do, but I think adding an exclusivity check to error if both models are set would be good.

Reply:
added a warning for this case.